packages/compass-app-stores/package.json

```diff
@@ -6,13 +6,12 @@
     "name": "MongoDB Inc",
     "email": "compass@mongodb.com"
   },
-  "private": true,
   "bugs": {
     "url": "https://jira.mongodb.org/projects/COMPASS/issues",
     "email": "compass@mongodb.com"
   },
   "homepage": "https://github.com/mongodb-js/compass",
-  "version": "0.5.2",
+  "version": "7.0.0",
```

Comment:
This was incorrectly versioned and marked as private. Fixed, version is just next major from what was last published